modules/flowable-engine-common-api/src/main/java/org/flowable/common/engine/api/tenant/ChangeTenantIdResult.java

```diff
@@ -0,0 +1,86 @@
+package org.flowable.common.engine.api.tenant;
```

Comment:
Looks like you missed one; the license header is missing here.

Reply:
Missing License header added. 👍🏻